### PR TITLE
[Add] Add WebSocket sample for Meta, Quote and Chart to the sample app.

### PR DESCRIPTION
**Pull request by nick45chen**
## Description

新增 WebSocket 示範資料

### CODE REVIEW

**Comment by nick45chen** [comment]
電腦網路有問題，待測試 🙏

File: app/src/main/java/com/example/fugle_realtime_kotlin_sample/data/WebSocketData.kt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    @OptIn(DelicateCoroutinesApi::class)
+    object Quote : WebSocketData() {
+
+        override fun name(): String = "Quote"
+
+        override fun start(symbolId: String, token: String, callback: (String) -> Unit) {
+            callback.invoke("InProgress")
+            GlobalScope.launch {
+                val flow = webSocketClient.connectQuote("2884", "demo")
```
Comment (dan12411) [inline app/src/main/java/com/example/fugle_realtime_kotlin_sample/data/WebSocketData.kt:46]: nitpick: "2884", "demo" 這兩個應可抽出成共用的參數 🙋🏼

**Review by dan12411 — APPROVED** [review]

File: app/src/main/java/com/example/fugle_realtime_kotlin_sample/data/WebSocketData.kt (hunk 1 above)
Comment (nick45chen) [reply app/src/main/java/com/example/fugle_realtime_kotlin_sample/data/WebSocketData.kt]: 感謝 @dan12411 已調整

https://github.com/Make-a-good-soup/fugle-realtime-kotlin/pull/23/commits/551a47720cb8d59443bd82de4540eb7e2b1be06b

**Review by dan12411 — APPROVED** [review]
LGTM!

**Pull request merged**